### DISCUSSION TITLE
Замена виньеток

### DISCUSSION THREAD

**Discussion by NovomirK** [petrovvlad/freeLib#186] [Q&A] [answered]
Добрый день. 
Постольку в программе всё прекрасно работает, а руки чешутся решил поэстетствовать и поставить другие виньетки. Нашёл на известном форуме, скачал, скопировал с заменой нужные рисунки в папку пользователь/freeLib/build/xsl/img/ (kubuntu, программа установлена из исходников), а в книгах всё равно откуда-то подтягиваются старые. Помогите, пожалуйста, понять где и что менять. Спасибо!

**Comment by petrovvlad** [comment, marked as answer]
После компиляции изображения виньеток хранятся в исполняемом файле. Сейчас менять виньетки надо до компиляции. Может быть добавлю возможность использовать пользовательские виньетки.

**Reply by NovomirK** [reply]
Если когда-нибудь дойдут руки, то будет интересно поиграться с ними. Спасибо большое за ответ.
